### DISCUSSION TITLE
Dudas Sección pastelero

### DISCUSSION THREAD

**Discussion by omarchavez18** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#203] [Q&A]
Buenas tardes compañeros una pregunta la sección del pastelero que pueda visualizar los sabores restante así como los adornos, cómo puede hacerse ya que para que se tome la información del cliente y se pueda revisar cuál está libre sería con backend, cómo puede hacerse sin back end?? Gracias

**Comment by AngelCruzO** [comment]
De forma estatica, solo pon información inventada. Aunque algunos explorers te diran que con javascript de forma local, puedes hacer dinamica la información. Pero desde mi punto de vista, si no tienes mucha  experiencia o no la tienes, es mejor solo hacer la maquetación HTML o ponerla bonita ya con CSS

**Reply by omarchavez18** [reply]
muchas gracias

**Comment by hackit92** [comment]
Si no tienes mucha experiencia podrías hacer uso de una tabla y algunos otros elementos para crear un dashboard estático, ya si te quieres aventurar al backend sería avanzar una buena parte del siguiente módulo... siempre es bueno ser curiosos jeje

**Reply by omarchavez18** [reply]
muchas gracias

**Comment by linettelopb** [comment]
Al igual que mis compañeros concuerdo que si no tienes tanta experencia solo utiliza datos estaticos, ya que no es necesario que tenga funcionalidad lo importante de esta fase es la maquetación y los  estilos. En mi opinión adelantarte a algo  mas complejo que problamente te enrrede mas es innecesario y ya que por lo visto se van a estar cambiando de ejemplos ya que en las practicas cambian y en los lives tambien. 
Queda a criterio propio pero no esta mal llevar el ritmo de lo indicado o un poco mas.

**Reply by omarchavez18** [reply]
muchas gracias linette

**Comment by JosePabloGrr** [comment]
1. En todo proyecto como front-end no siempre te proporcionan APIs (a veces tu vas adelantando lo visual) en este caso necesitas utilizar Mockup de información (tablas con contenido genérico o estático)
2. Si quieres aparentar que tiene función puedes utilizar funciones de javascript (sin backend) donde al dar clic en un botón como "Ver más" ejecutes una función con Append al contenido de dicha tabla :) (para fingir un Ajax Request, agregar y ver más contenido)

De momento no te estreses con backend, solo necesitaban lo visual, la funcion de los botones y demás se vera en backend.. 👍🏻

**Comment by JuanSalvador73** [comment]
Haz la parte del formulario con "checkbox" para que el cliente pueda escoger los sabores o adornos que desea. Por lo demás, solo haz el HTML así solo, estático. De momento no se requiere que sea dinámico, eso se verá mas adelante.

Algunos explorers ya han publicado sus ejemplos de sitios de pasteles con host gratis, chécalos, son muy buenos e ilustrativos. Solo recuerda que de momento con que tengas el HTML es suficiente, si hay algún aspecto técnico que sientes que te falta déjalo para adelante, insisto, este sitio es por etapas seguramente.

**Comment by HeribertoBonilla** [comment]
Wow me parece genial lo que estas haciendo :)
